src/containers/ViewerPanel/index.tsx

```diff
@@ -234,32 +238,24 @@ class ViewerPanel extends React.Component<ViewerPanelProps, ViewerPanelState> {
 
     public onTrajectoryFileInfoChanged(data: TrajectoryFileInfo) {
         const { receiveMetadata, simulariumController } = this.props;
-        const { spatialUnitFactorMeters } = data;
-
+        const updatedData = data as TrajectoryFileInfoV2;
```

Comment:
I'm confused by this typing, with the updated version, shouldn't `onTrajectoryFileInfoChanged` already be typed to the most recent version. Seeing this here also makes me think that type `TrajectoryFileInfo` should be the current version and the only type that the front end ever needs to use, and the viewer can set `TrajectoryFileInfo = TrajectoryFileInfoV2` or whatever is the current version, but it shouldn't be exporting `TrajectoryFileInfoV2`

Reply:
Yeah, I agree with you and @toloudis that there shouldn't be references to `TrajectoryFileInfo` AND `TrajectoryFileInfoV2` in simularium-website. I don't think I can use `TrajectoryFileInfo` here though, because that's defined as `TrajectoryFileInfoV1 | TrajectoryFileInfoV2` in the viewer type definitions. 

Instead I can make a Jira ticket to have [`updateTrajectoryFileInfoFormat` in simularium-viewer](https://github.com/allen-cell-animated/simularium-viewer/blob/master/src/simularium/versionHandlers.ts#L8) output a `TrajectoryFileInfoV2` instead of `TrajectoryFileInfo`. After that, I can modify simularium-website to only use `TrajectoryFileInfoV2`. Yes it has a version number and yes it'll need to be changed in name when we go to V3 but I think that makes sense since it would reflect a change in the shape of the incoming data. What do you guys think of that?

Reply:
I think the front end should never have to change the import.... the viewer should export something like `CurrentFileInfoFormat` that can be set to whatever the current one is. So in the viewer it would be `export type CurrentFileInfoFormat = TrajectoryFileInfoV2` and you would change that to `export type CurrentFileInfoFormat = TrajectoryFileInfoV3` on the viewer when the time comes

Reply:
Oh I see what you're saying. Can do

Reply:
Made a ticket: https://aicsjira.corp.alleninstitute.org/browse/AGENTVIZ-1382